File: dayFour/javaScript/taskTwo.js
let total = 0;

for(let count = 1; count <= 10; count++){
    
    const scanner = require("prompt-sync")();
    let number = Number(scanner ("Enter score: "));
    total = total + number;
}
let average = total / 10;

console.log("The average of the ten scores is "+average);

File: dayFour/java/afterBreak/TaskTwo.java
import java.util.Scanner;

public class TaskTwo{

    public static void main(String[] args){
    
        Scanner input = new Scanner(System.in);
        int total = 0;
        
        for(int count = 1; count <= 10; count++){
            System.out.println("Enter a score: ");
            int number = input.nextInt();
            total += number;
        }
        int average = total / 10;
        System.out.println("The average of the ten scores is "+average);
    }
}
